config/settings.yml

```diff
@@ -1480,6 +1480,6 @@
       :memory_threshold: 1.gigabytes
       :nice_delta: 1
     :websocket_worker:
-       :connection_pool_size: 8
+       :connection_pool_size: 26
```

Comment:
@himdel has a high-prio BZ for this, I am pretty sure that the websocket worker can eat up to 26 threads, maybe even 27 but I can't prove it in a real case scenario :(